packages/ui/src/app/components/PageLayout.tsx

```diff
@@ -16,15 +16,19 @@ export interface PageLayoutProps {
 export const PageLayout = ({ header, main, sidebar, footer, lastBreadcrumb }: PageLayoutProps) => (
   <PageContent>
     <Breadcrumbs lastBreadcrumb={lastBreadcrumb} breadcrumbsOptions={breadcrumbsOptions} />
-    <PageLayoutComponent header={header} main={main} sidebar={sidebar} footer={footer}>
-      {header && <PageHeader>{header}</PageHeader>}
-      {main && <PageMain>{main}</PageMain>}
-      {sidebar && <PageSidebar>{sidebar}</PageSidebar>}
-      {footer && <PageFooter>{footer}</PageFooter>}
-    </PageLayoutComponent>
+    <PageLayoutContent header={header} main={main} sidebar={sidebar} footer={footer} />
```

Comment:
I think the way you implemented it is fine. I would have probably done something like:
```typescript
export const PageLayout = ({ header, main, sidebar, footer, breadcrumbs, lastBreadcrumb }: PageLayoutProps) => (
  <PageContent>
    {breadcrumbs ?? <Breadcrumbs lastBreadcrumb={lastBreadcrumb} breadcrumbsOptions={breadcrumbsOptions} />}
```
and (eg in `pages/Forum/ForumThread.tsx`):
```typescript
return <PageLayout breadcrumbs={<ThreadBreadcrumbs />} header={displayHeader()} main={displayMain()} sidebar={displaySidebar()} />
```
Too avoid adding `<ForumPageLayout />` but it could just be personal preference.

Reply:
I considered it originally, but considering how these custom breadcrumbs are only used in two forum pages, I decided to keep the site-wide breadcrumbs as a relatively simple component they were before.